test/parallel_api/ranges/merge_ranges_sycl.pass.cpp

```diff
@@ -0,0 +1,57 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/pstl_test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+
+#if _ONEDPL_USE_RANGES
+#include _PSTL_TEST_HEADER(ranges)
+#endif
+
+#include "support/utils.h"
+
+#include <iostream>
+
+int32_t
+main()
+{
+#if _ONEDPL_USE_RANGES
+    const int max_n = 10;
+    int data1[max_n] = { 0,  1, 2, 3, 4, 5, 6, 6, 6, 6};
+    int data2[max_n] = {-2, -1, 2, 3, 3, 5, 6, 7, 8, 9};
+    int data3[max_n] = { 0,  0, 0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
```suggestion
    int data3[2 * max_n] = {};
```
This will initialize `data3` with zeroes (https://en.cppreference.com/w/cpp/language/zero_initialization: _If T is array type, each element is zero-initialized_). We need `2 * max_n` for merging.